### ISSUE TITLE
Robot is just Bouncing in Gazebo 

### ISSUE THREAD

**Issue by ShubhamNandi** [chvmp/champ#68]
Hello I'm linking the [Repo](https://github.com/ShubhamNandi/dd_dot-
![dd_dot]) 
![dd_dot](https://user-images.githubusercontent.com/28607689/134642142-72bd0e30-7b03-44fc-8918-2e3b549ae3ac.gif)
I have made this robot model with [Fusion2Urdf](https://github.com/syuntoku14/fusion2urdf)
Please let me know what have i missed!

**Comment by RamCharanThota** [comment]
Even I faced the similar issue. check the PID values in control.yaml file inside the your robot config package. Especially start with small P values.

**Comment by abdu7rahman** [comment]
any update on how you managed to solve this issue?

**Comment by ShubhamNandi** [comment]
Yes, so the issue is with the PID. This time, I didn't change the PID and I'm using the default values. It's at least not exhibiting the same behavior, but I'm facing new issues now
Following is the error msg 
"TF_OLD_DATA ignoring data from the past for frame rf_shoulder_1 at time 38.73 according to authority unknown_publisher
"
